src/main/java/com/teenthofabud/wizard/nandifoods/wms/settings/error/SettingsExceptionHandler.java

```diff
@@ -0,0 +1,20 @@
+package com.teenthofabud.wizard.nandifoods.wms.settings.error;
+
+import com.teenthofabud.wizard.nandifoods.wms.error.WMSExceptionHandler;
+import com.teenthofabud.wizard.nandifoods.wms.error.core.WMSBaseException;
+import com.teenthofabud.wizard.nandifoods.wms.settings.unit.uom.error.UOMException;
+import com.teenthofabud.wizard.nandifoods.wms.settings.unit.vo.ErrorVo;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class SettingsExceptionHandler extends WMSExceptionHandler {
```

Comment:
Create a single global exception handler for all domain, sub domain specific exceptions